From 4e56b314424ed6d039f20194fc36eb89409f4977 Mon Sep 17 00:00:00 2001
From: Evan Green <evgreen@chromium.org>
Date: Mon, 27 Jul 2020 14:54:39 -0700
Subject: [PATCH] Fix modem configuration

This change, suggested by bjorn.andersson@linaro.org, makes data flow
work properly on the modem. From what I understand, the configuration
that the IPA kernel driver sets up with the modem needs to match the
configuration netmgr asks for. This change makes that configuration
match.

Signed-off-by: Evan Green <evgreen@chromium.org>
---
 src/mmdata_qrtr.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mmdata_qrtr.c b/src/mmdata_qrtr.c
index 79587dd..5c39eae 100644
--- a/src/mmdata_qrtr.c
+++ b/src/mmdata_qrtr.c
@@ -359,8 +359,8 @@ int mmdata_fmt_cfg(void)
   int ret = MMDATA_FAILURE;
   unsigned char fmt_cfg_req[]= { 0x00, 0x03, 0x00, 0x20, 0x00, 0x2E, 0x00, 0x11,
                             0x04, 0x00, 0x02, 0x00, 0x00, 0x00, 0x12, 0x04,
-                            0x00, 0x05, 0x00, 0x00, 0x00, 0x13, 0x04, 0x00,
-                            0x05, 0x00, 0x00, 0x00, 0x15, 0x04, 0x00, 0x01,
+                            0x00, 0x08, 0x00, 0x00, 0x00, 0x13, 0x04, 0x00,
+                            0x08, 0x00, 0x00, 0x00, 0x15, 0x04, 0x00, 0x01,
                             0x00, 0x00, 0x00, 0x16, 0x04, 0x00, 0xE0, 0x05,
                             0x00, 0x00, 0x17, 0x08, 0x00, 0x04, 0x00, 0x00,
                             0x00, 0x01, 0x00, 0x00, 0x00 };
-- 
2.26.2

